C++/include/data_structures/dynamic_array.hpp

```diff
@@ -0,0 +1,129 @@
+#include <cstdlib> // std::malloc(), std::realloc()
+
+/*
+	Author
+	------
+	Ryan Young (github.com/ryoung9550)
+
+	Brief
+	-------
+	Dynamic Array is an implementation from scratch based
+	off of the STL std::vector
+*/
+
+template<class T>
+class DynamicArray
+{
+private:
+	unsigned arrSize;
+	unsigned numOfItems;
+	T* itemList;
```

Comment:
- Use `size_t` instead of `unsigned`, as used in the STL.
- Please rename these data members so that they closely mimic those in the STL. Eg: `arrSize` can be renamed as `capacity` and `numOfItems` can be renamed as `size`. `itemList` can be renamed to just `items`.